lib/tasks/publishing_api.rake

```diff
@@ -146,6 +149,42 @@ namespace :publishing_api do
     end
   end
 
+  desc "Manually redirect already unpublished Statistics Announcements"
+  # Statistics Announcements are not the same as other documents - once unpublished, they disappear for the user,
+  # meaning users are unable to set different redirects or reasons for the removed statistics announcement
+  task :redirect_unpublished_statistics_announcement, %i[slug alternative_url locale] => :environment do |_, args|
+    args.with_defaults(locale: "en")
+
+    results = StatisticsAnnouncement.unscoped.where(slug: args[:slug])
+    if results.empty?
+      puts "Could not find Statistics Announcement with slug #{args[:slug]}"
+      next
+    end
+    if results.count > 1
+      puts "More than one Statistics Announcement with slug #{args[:slug]}"
```

Comment:
Well - i'm guessing because they are "unpublished" it's highly likely - as publishers might create something `slug-123` and decide to unpublish it, it disappears from Whitehall, and then they change their mind and create a new one with the same name. I don't know if Statistics Announcement slug generation would take into account `unpublished` ones for slug check as it's not part of the default scope. 

In either case, there is no uniqueness constraint on the DB and the tests did allow me to create multiple unpublishings of the same slug so... 🙈

Reply:
I'll make it clearer with the message

Reply:
Oh, I see. Thanks for explaining